types/time.go

```diff
@@ -1272,7 +1729,8 @@ func parseTime(sc *stmtctx.StatementContext, str string, tp byte, fsp int, isFlo
 
 // ParseDatetime is a helper function wrapping ParseTime with datetime type and default fsp.
 func ParseDatetime(sc *stmtctx.StatementContext, str string) (Time, error) {
-	return ParseTime(sc, str, mysql.TypeDatetime, GetFsp(str))
+	//return ParseTime(sc, str, mysql.TypeDatetime, GetFsp(str))
```

Comment:
Use delete rather than comment, git would keep the history.

Reply:
ok.I delete parseDatetime,and rename parseDatetimeV2 to parseDatetime.